lib/saml_idp/request.rb

```diff
@@ -75,6 +75,14 @@ def requested_authn_context
       end
     end
 
+    def requested_authn_context_comparison
+      if authn_request? && requested_authn_context_node
+        requested_authn_context_node["Comparison"]
+      else
+        nil
```

Comment:
can remove this since a falsy `if` block evaluates to `nil` anyways

```suggestion
```

Reply:
done, also cleaned up the method below it. Thanks for the suggestion!